LykkeApi2/Controllers/OrdersController.cs

```diff
@@ -266,7 +270,82 @@ public async Task<IActionResult> PlaceLimitOrder([FromBody] LimitOrderRequest re
             }
 
             return Created(Url.Action("Get", "Operations", new { id }), id);                  
-        }        
+        }
+        
+        [HttpDelete("limit/{orderId}")]
+        [SwaggerOperation("CancelLimitOrderNew")]
+        [ProducesResponseType(typeof(void), (int) HttpStatusCode.OK)]
+        [ProducesResponseType(typeof(void), (int) HttpStatusCode.NotFound)]
+        public async Task<IActionResult> CancelLimitOrderNew(string orderId)
+        {
+            var tradingSession = await _clientSessionsClient.GetTradingSession(_lykkePrincipal.GetToken());
+
+            var confirmationRequired = _baseSettings.EnableSessionValidation && !(tradingSession?.Confirmed ?? false);
+            if (confirmationRequired)
+            {
+                return BadRequest("Session confirmation is required");
+            }
+
+            var clientId = _requestContext.ClientId;
+
+            var activeOrders = await _limitOrdersRepository.GetActiveByClientIdAsync(clientId);
+
+            if (activeOrders.All(x => x.Id != orderId))
+                return NotFound();
+            
+            await _limitOrdersRepository.CancelByIdAsync(clientId, orderId);
```

Comment:
There is no need to call it here. TxHandler will do the same things after ME executes the command.